### DISCUSSION TITLE
Currently, EFCore7 already supports powerful Json operation and batch processing capabilities, and is there any plan to use the new technology on some functions?

### DISCUSSION THREAD

**Discussion by vipwan** [OrchardCMS/OrchardCore#12431] [Ideas]
Currently, EFCore7 already supports powerful Json operation and batch processing capabilities, and is there any plan to use the new technology on some functions 

I know Yessql and EF7 can coexist, but it's going to be weird :)

[EF7 whatsnew](https://docs.microsoft.com/zh-cn/ef/core/what-is-new/ef-core-7.0/whatsnew#json-columns)

**Comment by hishamco** [comment]
I already started working on a local branch ro support EF Core in Orchard Core Contrib (OCC), we could write some modules make use of it

For now YesSQL is the default implementation that OrchardCore comes with, but nothing stop us to use EF Core

**Comment by hishamco** [comment]
For reference:

https://github.com/OrchardCoreContrib/OrchardCoreContrib/issues/16
https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/issues/70

**Comment by Skrypt** [comment]
I think he is referring to supporting JSON functions inside YesSQL dialects. Which is something I've discussed with @sebastienros a while ago.

**Reply by hishamco** [reply]
But this leads to coexist of both YesSQL & EF Core

**Reply by vipwan** [reply]
The result I expect is unified to EFCore. Of course, this is a bit extravagant, after all, OC is too deeply integrated with Yessql

**Reply by hishamco** [reply]
That's what I refered too previously, we can develop some modules that are using EF Core, similar to what I wanna do in OCC.

OC modules is tightly coupled on YesSQL
